source/util/bit_stream.h

```diff
@@ -362,8 +404,14 @@ class BitReaderWord64 : public BitReaderInterface {
   // Consuming the original buffer and casting it to uint64 is difficult,
   // as it would potentially cause data misalignment and poor performance.
```

Comment:
More than just poor performance:  It would be invalid on some architectures. (SPARC would trigger a bus error)